src/borg/borg-think.c

```diff
@@ -365,7 +365,9 @@ bool borg_think(void)
 
     /* Hack -- always revert shapechanged players to normal form */
     if (player_is_shapechanged(player)) {
-        borg_keypress('m');
+        /* it looks like inscribe is a good command that checks */
+        /* your form without a prerequisite check */
+        borg_keypress('{');
```

Comment:
'q', for drinking a potion, may be better as there's been some discussion, but nothing done so far, about making dropping (and likely changing inscriptions) available when shapechanged:  for instance, see https://angband.live/forums/forum/angband/variants/248837-faangband-nightly-crashes-on-game-resumption?p=248840#post248840 .

Reply:
> 'q', for drinking a potion, may be better as there's been some discussion, but nothing done so far, about making dropping (and likely changing inscriptions) available when shapechanged: for instance, see https://angband.live/forums/forum/angband/variants/248837-faangband-nightly-crashes-on-game-resumption?p=248840#post248840 .

my worry with "q" was that it might change to check if there are potions in inventory before giving the change form option whereas inscribe is always an option as long as you have something in inventory or equipment.   Perhaps drop?

Reply:
Throw should be the most future-proof.